style.css

```diff
@@ -343,9 +343,159 @@ section#grave-care {
 
 .grave-care-overlay {
   background-color: rgba(0, 0, 0, 0.72);
-  position: absolute;
   width: 100%;
   height: 100%;
   padding: 5rem 15rem;
   text-align: center;
 }
+
+/*---Section Documents---*/
+
+section#speaker {
+  background-color: #fff;
+  display: flex;
+  padding: 60px 0px;
+}
+
+/*---Section Contact---*/
+
+section#contact {
+  padding: 60px 0px;
+}
+
+h3.contact-header {
+  text-align: left;
+  color: #fff;
+  font-size: 3.5rem;
+  margin-bottom:4rem;
+  font-family: var(--header-font);
+}
+
+#contact-container {
+  max-width: 1140px;
+    min-height: 130px;
+    display: block;
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+
+
+#contact-two-columns {
+  display: flex;
+}
+
+#contact-left {
+  width:35%;
+  padding-right: 10%;
+}
+
+#contact-right {
+  width:55%;
+}
+
+h4.contact {
+ 
+    color: #fff;
+    font-size: 1.1rem;
+    line-height: 2rem;
+font-weight:600;
+margin-bottom: 1.5rem;
+}
+
+p.contact-text {
+  color: #fff;
+  font-size: 1.1rem;
+  line-height: 2rem;
+  margin-bottom: 3rem;
+}
+
+p.contact-text-small {
+  color: #fff;
+  font-size: 0.9rem;
+  line-height: 1.5rem;
+  margin-bottom: 4rem;
+}
+
+
+h4.contact-gold {
+ 
+  color: var(--gold-color);
+  font-size: 1.3rem;
+  line-height: 3rem;
+font-weight:600;
+margin-bottom: 1.5rem;
+}
+
+a.contact-text-gold {
+color: var(--gold-color);
+font-size: 2rem;
+line-height: 2rem;
+margin-bottom: 2rem;
+}
+
+#map {
+  margin-bottom: 2rem;
+}
+
+
+
+
+
+/* ---Footer-links--- */
+
+#footer-links {
+width: 1140px;
+margin: auto;
+display:flex;
+
+}
+
+.three-columns {
+  width:30%;
+  float: left;
+  text-align: center;
+}
+
+.three-columns:nth-child(3) {
+  padding: 0 5%;
```

Comment:
Why do we need % values here? :)

Reply:
We don't need this.

Reply:
:) Percents are sometimes tricky, especially when working with different viewports. However in some situations they are best indeed. 

By the way, do you know of vw/vh units? Not for this case, but they are sometimes very helpful. :)
[CSS Units: vh, vw, vmin, vmax](https://www.youtube.com/watch?v=IWFqGsXxJ1E&ab_channel=KevinPowell)

Reply:
I'm using 100vh in this project in header. I'll read about vmin and vmax and maybe use it in next project :)

Reply:
Wonderful! You rock! 👯 🚀